front-end/Components/MatchDetailScreen.js

```diff
@@ -0,0 +1,78 @@
+//install MapView from react-native-maps
```

Comment:
Would you mind adding this to the Package JSON - just be good to keep GitHub nice and tidy :)